test/Feature/RpcSpec.hs

```diff
@@ -291,6 +291,20 @@ spec actualPgVersion =
             [json|"Hi"|]
             { matchHeaders = [matchContentTypeJson] }
 
+      it "works with inout argument" $
+        post "/rpc/inout_argument"
```

Comment:
I just realized that there are already `INOUT` tests in place. I guess I was searching for "argument" but those are labeled with "param", so I didn't find them. So this test can probably be removed.